javascript/javascript3/week3/ShoppingCart/shoppingcart-.js

```diff
@@ -0,0 +1,140 @@
+const body = document.getElementsByTagName("body");
+const h1 = document.getElementById("whos-shopping-cart");
+const itemList = document.getElementById("item-list");
+// const itemAddBtn = document.getElementById("add-btn");
+const cartTotal = document.getElementById("cart-total");
+const searchInput = document.getElementById("product-search-input");
+
+let cartTotalPrice = 0;
+
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+
+  convertToCurrency(currency) {
+    // Assuming dkr as default currency
+    try {
+      fetch(
+        `https://v6.exchangerate-api.com/v6/a534b80e56a63f91e332441c/latest/DKK`
+      )
+        .then((response) => response.json())
+        .then((data) => {
+          console.log(data);
+          const allCurrencies = data.conversion_rates[currency];
+          console.log(allCurrencies);
+          const changeMoney = this.price * allCurrencies;
+          console.log(Math.ceil(changeMoney) + " won");
```

Comment:
and why is it "won" here? Inside the function you don't know what the caller sent you.

Reply:
It was for testing, forgot to delete